rpc/block.go

```diff
@@ -61,7 +61,7 @@ func (provider *Provider) BlockWithTxHashes(ctx context.Context, blockID BlockID
 
 	// if header.Hash == nil it's a pending block
 	if result.BlockHeader.BlockHash == nil {
-		return PendingBlock{
+		return PendingBlockTxHashes{
```

Comment:
We return PendingBlockTxHashes (by value) here but a pointer (reference) in line 72. We should be consistent